src/plugins/sqlalchemy_create_object_operator.py

```diff
@@ -104,7 +136,7 @@ def execute(self, context: Dict[str, Any]) -> None:
         else:
             self.data_table_name = self.data_table_name
 
-        data_schema_url = SCHEMA_URL / self.data_schema_name / self.data_schema_name
+        data_schema_url = self.get_schema(self.data_schema_version)
```

Comment:
Ik denk dat we de functionaliteit van `get_schema` uiteindelijk  beter in schematools kunnen onderbrengen. Als we de functie `schema_fetch_url_file` robuuster maken zodat deze ook met versies om kan gaan, dan kunnen alle gebruikers van deze functie hier direct van profiteren.

Reply:
+1.
Uiteindelijk we hebben `schematools.utils.def_from_url` die gebruikt index om juiste versie van dataset (onafhankelijk van prefix) te vinden:
https://github.com/Amsterdam/schema-tools/blob/63a3327140f0040281ec0121f37bb19b1f94eed5/src/schematools/utils.py#L87

Reply:
Was look at PR https://github.com/Amsterdam/dataservices-airflow/pull/252 and now I indeed think this functionality should be in `schematools.utils.schema_def_from_url`

Reply:
@ngaranko you are right, `schema_def_from_url` is already taking versioning into account.

Reply:
Check, heb dit ook met Jan besproken. De logica voor ophalen juiste URL versie zal ik proberen om in schema-tools neer te zetten als centrale plek